client/bindings/nodejs/test/client/offlineSigningExamples.spec.ts

```diff
@@ -114,3 +113,15 @@ describe('Offline signing examples', () => {
         });
     });
 });
+
+// Convert a hex string to a byte array
+function hexToBytes(hex: string) {
```

Comment:
isnt there a method for this? for example https://stackoverflow.com/a/36050152/4512850
Seems silly to implement ourselves